accelerate/nuxtjs-starter/components/Quote.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+	<QuoteWrapper :title="title" :type="type">
+		<div class="flex flex-col">
+			<p v-if="result?.data" class="text-lg">
+				<span class="text-green-300">ID {{ id }}</span> ⸺ '"
+				<span v-html="quote"></span>
+				"'
+			</p>
+			<br v-if="result?.data" />
+			<p v-if="result?.data" class="text-lg font-normal text-black ">
+				<span class="mt-3.5">Created At</span> ⸺
+				{{ formatDate(createdAt) }}
+			</p>
+			<div v-if="result?.data" class="my-2 h-0.5 border-t-0 bg-neutral-100 opacity-100 opacity-50"></div>
+			<div v-if="result?.data">
+				<br />
+				<p>
+					Cache Node Region ⸺
+					<span class="font-bold">
+						{{ findIATA(region)?.city || region }}
+					</span>
+				</p>
+				<br />
+				<p>
+					Cached Modified at ⸺
+					<span class="font-bold">
+						{{ formatDate(lastModified) }}
+					</span>
+				</p>
+				<br />
+				<p>
+					Cache status ⸺
+					<span class="font-bold" :class="{
+						'text-green-400': cacheStatus === 'swr' || cacheStatus === 'ttl',
+						'text-red-400': cacheStatus !== 'swr' && cacheStatus !== 'ttl'
+					}">
+						{{ cacheStatus?.toUpperCase() }}
+						{{ cacheStatus === 'swr' || cacheStatus === 'ttl'
+							? ' CACHE HIT'
+							: '' }}
+					</span>
+				</p>
+				<br />
+				<p>
+					Time taken: <span class="font-bold"> {{ result.time }}ms</span>
+				</p>
+				<br />
+			</div>
+			<p v-else class="text-red-500">No data available</p>
```

Comment:
I see that it also shows "no data available" when we reload the page. Can we change it to something like "Loading component" when it's actually loading?

And if there is no data, then show no data is available.

Reply:
@ludralph